.github/workflows/publish_docker.yaml

```diff
@@ -62,21 +56,6 @@ jobs:
             echo "match=false" >> $GITHUB_OUTPUT
           fi
 
-      # if neither prod, rc, beta or alpha git tag, then push images with the ":dev" tag
-      - name: Push Fides Dev Tag
-        if: steps.check-prod-tag.outputs.match == 'false' && steps.check-rc-tag.outputs.match == 'false' && steps.check-beta-tag.outputs.match == 'false' && steps.check-alpha-tag.outputs.match == 'false'
-        run: nox -s "push(dev)"
-
-      # if a prod git tag, then we run the prod job to publish images tagged with the version number and a constant ":latest" tag
-      - name: Push Fides Prod Tags
-        if: steps.check-prod-tag.outputs.match == 'true'
-        run: nox -s "push(prod)"
-
-      # if an RC git tag, then we run the rc job to publish images with an ":rc" tag
-      - name: Push Fides RC Tags
-        if: steps.check-rc-tag.outputs.match == 'true'
-        run: nox -s "push(rc)"
-
       # if an RC git tag, also notify Fidesinfra to trigger a redeploy of rc env, to pick up our newly published images
       - name: Send Repository Dispatch Event (RC redeploy)
```

Comment:
whoooops, i missed this...this needed to move down below, it needs to run _after_ the `Push` job completes. i'll create a follow-up.

Reply:
here's a follow up/fix: https://github.com/ethyca/fides/pull/4017